### PR TITLE
Clarify rocket docs

### PR DESCRIPTION
**Pull request by SarahBornais**
Remove the dollar signs in the instructions for Rocket setup and provide examples of the Rocket commands

### CODE REVIEW

File: handbook/onboarding/leads.md [hunk 1]
```diff
@@ -84,9 +82,17 @@ Team formation happens in [kickoff events](/handbook/strategy/recurring-processe
...
+
+   ```
+   /rocket team create TEAM_NAME --lead SLACK_HANDLE --channel TEAM_CHANNEL
    ```
-   /rocket team create $TEAM --lead $YOU --channel $CHANNEL
+
+   For example, if your team name is `Facebook for Cats`, your Slack handle is `@Jane Doe`, and the team Slack channel you created in step 1 is called `#tm-facebook-for-cats`, then you should write:
+
    ```
+   /rocket team create "Facebook for Cats" --lead @Jane Doe --channel #tm-facebook-for-cats
```
Comment (wrenliang) [inline handbook/onboarding/leads.md:93]: Does the @Jane Doe work if there's a space? Do we also need to wrap that in quotations

**Review by wrenliang — APPROVED** [review]
Left a minor comment, otherwise looks good!

File: handbook/onboarding/leads.md (hunk 1 above)
Comment (SarahBornais) [reply handbook/onboarding/leads.md:93]: Yes, it does. I used a Slack handle with a space on purpose to illustrate this. Slack ends up recognizing it as a tag object like so: 
image

Comment (SarahBornais) [reply handbook/onboarding/leads.md:95]: Added note for clarity:
image

**Pull request merged**